README.md

```diff
@@ -14,20 +14,23 @@ This ckan extensions adds a user search and user display by organization and rol
 
 ## Update translations
 
-To generate a new ckanext-switzerland.pot file inside the Docker container,
-use the following command:
+To generate an updated ckanext-switzerland.pot file inside the Docker
+container, use the following commands:
 
     docker-compose exec ckan bash
     source /usr/lib/ckan/venv/bin/activate
-    cd /usr/lib/ckanext/ckanext-switzerland_users/
+    cd /usr/lib/ckanext/ckanext-switzerland-ng/
     python setup.py extract_messages
 
-This will generate the `pot` files
-Then update the `po` files with:
+Copy any new strings that you want to translate from the new
+`ckanext-switzerland.pot` into the `ckanext-switzerland.po` file for each
```

Comment:
Please correct the filenames to "ckanext-switzerland_users.po" and "ckanext-switzerland_users.pot"